sdk/framework/openapi.go

```diff
@@ -395,7 +391,7 @@ func documentPath(p *Path, specialPaths *logical.Paths, requestResponsePrefix st
 
 				// Set the final request body. Only JSON request data is supported.
 				if len(s.Properties) > 0 || s.Example != nil {
-					requestName := constructRequestResponseName(path, requestResponsePrefix, "Request")
+					requestName := constructRequestResponseIdentifier(opType, defaultMountPath, path, "request")
```

Comment:
do we want title or lower case?

Reply:
All parts are converted to title case in the output